examples/powercli/ha-restarted-vms/stack.yml

```diff
@@ -0,0 +1,17 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://veba.abc.com
+functions:
+  powercli-ha-restarted-vms:
+    lang: powercli
+    handler: ./handler
+    image: rkt6865/veba-powercli-ha-restarted-vms:latest
```

Comment:
Can you replace `rkt6865/veba-powercli-ha-restarted-vms` with `vmware/veba-powercli-ha-restarted-vms` and I'll go ahead and get that repo created and build this function officially

Reply:
Thanks William!  I just made the change.  I forgot to ask prior, but, do I need to squash it again?

Reply:
Yes, please squash the commit and we'll be good to merge